_posts/blog/2016-12-19-running-on-fedora-25.md

```diff
@@ -0,0 +1,335 @@
+`root` user, you'll have to set its password to the same one you'll use later
+when you initialize Accumulo. If you use another user name, you'll have to
+create that user later.
+
+### Configure Hadoop client
+
+Now, Hadoop's default local filesystem handler isn't very good at ensuring
+files are written to disk when services are stopped. So, let's use a better
+filesystem implementation for `file://` locations. Even though we're not
+running HDFS, the Hadoop client code used in Accumulo can still be configured
+by modifying Hadoop's configuration file:
+
+```bash
+sudo vim /etc/hadoop/core-site.xml
+```
+
+Add a new property:
+
+```xml
+  <property>
+    <name>fs.file.impl</name>
+    <value>org.apache.hadoop.fs.RawLocalFileSystem</value>
+  </property>
+```
+
+### Initialize Accumulo
+
+Now, initialize Accumulo. You'll need to do this as the `accumulo` user,
+because the accumulo services are configured to run as the `accumulo` user.
+This user is created automatically when you install the RPMs. If you already
+have a user and/or group by this name, it will probably not be a problem, but
+be aware that this user will have permissions for the server configuration
+files. To initialize as a specific user, use `sudo -u`:
+
+```bash
+sudo -u accumulo accumulo init
+```
+
+Like always, this command will fail if ZooKeeper is not running, or if the
+destination volume (`file:///var/tmp/accumulo`) already exists.
+
+### Start Accumulo services
+
+Now that Accumulo is initialized, you can start its services:
+
+```bash
+sudo systemctl start accumulo-{master,tserver,gc,tracer,monitor}.service
+```
+
+Enable the commands to start at boot:
+
+```bash
+sudo systemctl enable accumulo-{master,tserver,gc,tracer,monitor}.service
+```
+
+## Running multiple nodes
+
+### Amazon EC2 setup
+
+For a multi-node setup, we used a Fedora 25 Cloud AMI on Amazon EC2, HVM
+virtualization type, with gp2 disk type with the root volume on EBS. Launched
+using 64GB for the root volume and did not attach additional storage. The
+latest AMI, as of this writing, seems to be `ami-e5757bf2` (`us-east-1`
+availability zone). For this example, launch three instances (in the same
+security group). The default user is `fedora` for the Fedora AMIs. Don't forget
+to make sure they are fully up-to-date (`sudo dnf --refresh upgrade`).
+
+For this setup, you should have a name service configured properly. For
+convenience, we used the EC2 provided internal DNS, with internal IP addresses.
+Make sure the nodes can communicate with each other using these names. If
+you're using EC2, this means making sure they are in the same security group,
+and the security group has an inbound rule for "All traffic" with the source
+set to the security group (`sg-xxxxxxxx`).
+
+### Configure and run Hadoop
+
+Set up Hadoop HDFS. This is the primary place we differ from the single node
+setup. For both Hadoop and Accumulo, you can edit the configuration files on
```

Comment:
First 2 sentences could be combined... something like "Setting up Hadoop to use HDFS is the primary place we differ from single node setup"